server/rest/src/test/java/org/infinispan/rest/framework/RestDispatcherTest.java

```diff
@@ -36,53 +37,53 @@ public void testDispatch() throws Exception {
       RestDispatcherImpl restDispatcher = new RestDispatcherImpl(manager);
 
       RestRequest restRequest = new SimpleRequest.Builder().setMethod(GET).setPath("/ctx/").build();
-      RestResponse response = restDispatcher.dispatch(restRequest);
-      assertEquals("Hello World!", response.getEntity().toString());
+      CompletionStage<RestResponse> response = restDispatcher.dispatch(restRequest);
+      assertEquals("Hello World!", response.toCompletableFuture().get().getEntity().toString());
```

Comment:
You could use `CompletionStages.join()`, looks much better with a static import

Reply:
Indeed, thanks!